python/src/opendp/mod.py

```diff
@@ -492,6 +497,11 @@ def _depends_on(self, *args):
         """Extends the memory lifetime of args to the lifetime of self."""
         setattr(self, "_dependencies", args)
 
+    def __getattr__(self, name: str) -> Any:
+        from opendp.domains import _user_domain_descriptor
+        descriptor = _user_domain_descriptor(self)
+        return getattr(descriptor, name)
```

Comment:
```suggestion
        return getattr(self.descriptor, name)
```

Reply:
I also wonder if this is really a feature we want. It's a little bit of a convenience, but I could imagine it causing confusion: For example, if the user thinks they are accessing an attribute of `descriptor` but they actually get an attribute of this object.

Reply:
How do you feel about a shorter accessor instead?

```python
@property
def d(self):
    return self.descriptor
```

There's a lot of code that accesses attributes downstream: `input_domain.d.norm`, `input_domain.d.origin`, etc.

Reply:
I feel a little queasy about the single letter names that are already in the API, but that's just me.

For me, I'm less concerned about saving keystrokes and more interested in how we talk about the object. If this is just a shortcut I'm uncomfortable with it, but if `descriptor` is really an implementation detail we want to hide, then great, let's do that!